packages/manager/modules/cloud-connect/src/overview/datacenter/remove-extra/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,6 @@
+{
+  "cloud_connect_routing_remove": "Supprimer extra",
```

Comment:
Is this provided content ?

Reply:
We are yet to get proper messages and content